frontend/views/components/LinkToCopy.vue

```diff
@@ -47,20 +47,13 @@ export default ({
     }
   },
   methods: {
-    // FIXME: probably needs to be defined centrally somewhere else
-    isMacSafari () {
-      // NB: Chrome, Edge, Brave on Macintosh tacks on the 'Safari/537.36' string in its userAgent
-      // so we have to ensure that were not mistaking those browsers instead when matching for Safari on Macintosh
-      const userAgent = navigator.userAgent
-      const safari = /Macintosh.*Safari/
-      const chrome = /Macintosh.*Chrome/
-      const edge = /Macintosh.*Edg/
-      return userAgent.match(safari) &&
-              !(userAgent.match(chrome) || userAgent.match(edge))
+    isMobile () {
+      // https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser#comment95674193_51774045
+      return !window.matchMedia('(any-pointer:fine)').matches
```

Comment:
Since Travis is failing (not because of this PR, but because of a heisenbug), I'm thinking maybe we should make a change here anyway, a small optimization (and plus trigger a re-run of the Travis tests): this value doesn't change, right? It should stay the same for the entire session?

If so, then we can define `isMobile` as a `const` variable at the top of the file instead of defining it as a function. This way we don't have to run the test each time `copyToClipboard` is called.

Reply:
Good point!